packages/elements/src/components/ino-avatar/ino-avatar.scss

```diff
@@ -0,0 +1,156 @@
+@use '../base/typography-new' as typography;
+@use '../base/new-theme' as theme;
+
+/**
+  * @prop --ino-avatar-size: Width and height of the ino-avatar
+  */
+
+@mixin flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@keyframes rotate {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(60deg);
+  }
+}
+
+@keyframes rotateBack {
+  0% {
+    transform: rotate(60deg);
+  }
+  100% {
+    transform: rotate(0deg);
+  }
+}
+
+.ino-avatar {
+  --avatar-size: var(--ino-avatar-size, 138px);
+  @include flex-center;
+  width: var(--avatar-size);
+  height: var(--avatar-size);
+  color: theme.$p-5;
+  font-size: calc(var(--avatar-size) * 0.3);
+  font-weight: 600;
+  position: relative;
+
+  &--interactive {
```

Comment:
The interactive flag should also make the cursor a pointer to indicate some interactivity.